### PR TITLE
Updating debian/changelog for the 2.0 release

### PR DESCRIPTION
**Pull request by elopio**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,92 @@
...
+  * cli: strip command migrated to docopts.
+  * Obsolete deprecations from 1.x.
+  * Removing the 'vendor' keyword.
+  * Making the icon optional.
+  * cli: snap command migrated to docopts.
+  * Enabling testing with new cli.
+  * Fix ordering for conditional negations in catkin tests.
+  * Replace use of `assert_not_called` in the catkin tests.
+  * General documentation update.
+  * Add licensing support.
```
Comment (sergiusens) [inline debian/changelog]: This has a bug.

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,92 @@
...
+  * Remove dead code from lifecycle.
+  * cli: strip command migrated to docopts.
+  * Obsolete deprecations from 1.x.
+  * Removing the 'vendor' keyword.
+  * Making the icon optional.
+  * cli: snap command migrated to docopts.
+  * Enabling testing with new cli.
+  * Fix ordering for conditional negations in catkin tests.
+  * Replace use of `assert_not_called` in the catkin tests.
+  * General documentation update.
```
Comment (sergiusens) [inline debian/changelog]: This has a bug

File: debian/changelog [hunk 3]
```diff
@@ -1,3 +1,92 @@
...
+  * cli: init cmd migrated to docopts.
+  * cli: pull command migrated to docopts.
+  * clean cmd: migrate to use new lifecycle attribs.
+  * Removing dead test code.
+  * cli: build command migrated to docopts.
+  * cli: stage command migrated to docopts.
+  * Remove dead code from lifecycle.
+  * cli: strip command migrated to docopts.
+  * Obsolete deprecations from 1.x.
+  * Removing the 'vendor' keyword.
```
Comment (sergiusens) [inline debian/changelog]: This has a bug

File: debian/changelog [hunk 4]
```diff
@@ -1,3 +1,92 @@
...
+  * add missing 'to'.
+  * rephrase sentence based on Leo's feedback.
+  * remove vendor for 'master'.
+  * add document about parts.
+  * add link to config docs.
+
+  [ Sergio Schvezov ]
+  * Removing the `run` command.
+  * Update the release for the tomcat example snap.
+  * Wrapping exe should not be over match to replace.
```
Comment (sergiusens) [inline debian/changelog]: This has a bug. Also, fix my original typo and remove *be*

File: debian/changelog [hunk 5]
```diff
@@ -1,3 +1,92 @@
...
+  * Documentation refresh. (LP: #1533021)
+  * Support the new security-override semantics. (LP: #1533293)
+  * Remove ssh module from snapcraft. (LP: #1533400)
+  * Fix environment variable handling. (LP: #1531481)
+  * Wrap apps with the app name. (LP: #1534150)
+  * Replace SNAP envvars with the 16.04 ones. (LP: #1533552)
+
+  [ Leo Arias ]
+  * Added a test to check that the snapcraft files are not copied to the build dir.
+  * On autopkgtests, run the integration tests, build the examples but do not install them, do not run the unittests.
```
Comment (sergiusens) [inline debian/changelog]: This seems to be over 79 chars (which should raise a dh lint warning)

File: debian/changelog [hunk 6]
```diff
@@ -1,3 +1,92 @@
...
+  * Translated the plainbox tests to python.
```
Comment (sergiusens) [inline debian/changelog]: This had a bug iirc.

File: debian/changelog [hunk 7]
```diff
@@ -1,3 +1,92 @@
...
+  * Added the network-listener capability. (LP: #1534784)
+  * Add a security-override for chown.
+
+  [ Kyle Fazzari ]
+  * Catkin plugin: Add support for ROS tools.
+  * Add contribution guide.
+  * Resolve FIXME in Catkin plugin.
+  * Add a tutorial for turning a ROS project into a .snap.
+  * Catkin plugin: use rosdep for dependency resolution.
+  * Python{2,3} plugins: Make site-packages link relative.
```
Comment (sergiusens) [inline debian/changelog]: this has a bug

File: debian/changelog [hunk 8]
```diff
@@ -1,3 +1,92 @@
...
+  * Catkin plugin: Refactor build.
+  * ROS documentation: Fix missing step to make script executable.
```
Comment (sergiusens) [inline debian/changelog]: this has a bug

File: debian/changelog [hunk 9]
```diff
@@ -1,3 +1,92 @@
...
+  * Catkin plugin: Improve cmake path rewrite.
+  * Add support for mesa libraries. (LP: #1531620)
+  * Autotools plugin: Set autogen.sh executable. (LP: #1530995)
+  * Update CONTRIBUTING.md to require a bug. (LP: #1532195)
+  * Ensure rosdep resolves dependencies using Trusty. (LP: #1532241)
+  * Catkin plugin: Use in-snap python instead of OS-provided. (LP: #1533297)
+
+  [ Alexander Sack ]
+  * use archive.apache.org URL to ensure we refer to a stable URL that does not disappear on latest release.
```
Comment (sergiusens) [inline debian/changelog]: this has a bug, also check the length

**Comment by sergiusens** [comment]
Mind also adding http://paste.ubuntu.com/14545132/

File: debian/changelog (hunk 2 above)
Comment (elopio) [inline debian/changelog]: I can't find this one.
There is Documentation refresh. (LP: #1533021), already referenced to a doc bug. Maybe you were thinking about that one?

File: debian/changelog (hunk 6 above)
Comment (elopio) [inline debian/changelog]: I think not. I can't find it nor remember it.

File: debian/changelog (hunk 8 above)
Comment (elopio) [inline debian/changelog]: I can't find it.

File: debian/changelog (hunk 9 above)
Comment (elopio) [inline debian/changelog]: I can't find the bug.

**Comment by elopio** [comment]
docopt should be added in a different PR. But what the hell, lets get crazy today.

**Comment by elopio** [comment]
@sergiusens updated with all the bugs I could find.

File: debian/changelog (hunk 2 above)
Comment (sergiusens) [inline debian/changelog]: That might be the case. Maybe this line in the log is redundant.

**Comment by sergiusens** [comment]
@elopio can you merge once more and add that final commit?

File: debian/changelog (hunk 2 above)
Comment (elopio) [inline debian/changelog]: It comes from a different PR, so mmm, I don't know.
What I think is that from now on, we should make our commit messages better with more context for the changelog. More verbose.

**Comment by elopio** [comment]
@sergiusens done.

File: debian/changelog (hunk 2 above)
Comment (kyrofa) [inline debian/changelog]: Could not agree more. From now on let's make sure the PRs we review have meaningful commit messages.

File: debian/changelog [hunk 10]
```diff
@@ -1,3 +1,103 @@
+snapcraft (2.0) xenial; urgency=medium
+
+  * Updating debian/changelog for the 2.0 release
+
+ -- Leo Arias <yo@elopio.net>  Mon, 18 Jan 2016 11:59:28 -0600
```
Comment (kyrofa) [inline debian/changelog]: What is this entry?

File: debian/changelog [hunk 11]
```diff
@@ -1,3 +1,103 @@
...
+
+  [ Daniel Holbach ]
+  * Add document detailing the snapcraft.yaml syntax.
+  * Fix typo - thanks Kyle.
+  * Add missing periods.
+  * Add missing 'to'.
+  * Rephrase sentence based on Leo's feedback.
+  * Remove vendor for 'master'.
+  * Add document about parts.
+  * Add link to config docs.
```
Comment (kyrofa) [inline debian/changelog]: I'd clean some of these up. One of the reasons we squash nowadays-- no one cares about all these.

File: debian/changelog (hunk 10 above)
Comment (elopio) [inline debian/changelog]: grr, good catch. This is a leftover from a merge. Updating.

File: debian/changelog (hunk 11 above)
Comment (elopio) [inline debian/changelog]: So, just remove them? Or should I rephrase them somehow?
Rephrasing everything would be nice, but I can't remember what each entry is about to do it accurately.

File: debian/changelog [hunk 12]
```diff
@@ -1,3 +1,97 @@
...
+  [ Jonathan Cave ]
+  * Go plugin creating invalid env. (LP: #1524374)
+
+  [ Renat Galimov ]
+  * Close not called on the qml plugin. (LP: #1531994)
+
+  [ Vamshi Balanaga ]
+  * Added test for git sources in sources.py (LP: #1534411)
+
+ -- Leonardo Arias Fonseca <leo.arias@canonical.com>  Fri, 15 Jan 2016 17:52:01 -0600
```
Comment (sergiusens) [inline debian/changelog]: Add the 1.0 changelog here.

File: debian/changelog (hunk 11 above)
Comment (kyrofa) [inline debian/changelog]: Nahh, I wouldn't say to rephrase anything (so they can be matched up with commits), but things like "Fix typo - thanks Kyle." That's not helpful-- it's changelog noise. I'd remove things like that.

Comment (elopio) [inline debian/changelog]: Done.

File: debian/changelog (hunk 12 above)
Comment (elopio) [inline debian/changelog]: Feels weird, but done.

Comment (sergiusens) [inline debian/changelog]: yeah, this is why https://launchpad.net/ubuntu/+source/snapcraft/1.0

Comment (sergiusens) [inline debian/changelog]: In theory this is why we need to craft a changelog, create a packaging branch and keep CHANGELOG updated with commits on master and debian/changelog abstained to packaging bugs (this is the way it is originally intended).

**Comment by elopio** [comment]
\o/ I survived dch

**Pull request merged**